package.json

```diff
@@ -32,6 +32,7 @@
     "dotenv": "^16.4.5",
     "lucide-react": "^0.453.0",
     "mongodb": "^6.10.0",
+    "mongosh": "^2.3.3",
```

Comment:
Does prisma need this?

Reply:
, just a shell into Mongo. Compass has an interface to mongosh if one would need it